docs/tutorials/add-catalog.md

```diff
@@ -18,7 +18,6 @@ This catalog is distributed as an image [quay.io/operatorhubio/catalog](https://
 
 * Access to a Kubernetes cluster, for example `kind`, using an account with `cluster-admin` permissions
 * [Operator Controller installed](https://github.com/operator-framework/operator-controller/releases) on the cluster
-* [Catalogd installed](https://github.com/operator-framework/catalogd/releases/) on the cluster
```

Comment:
Idt this whole line should go away, catd is still needed on the cluster.

Reply:
We have 1 project now.
The release of https://github.com/operator-framework/operator-controller/releases will end up a file with both projects. So, I updated it for

* [Operator Controller and Catalogd installed](https://github.com/operator-framework/operator-controller/releases) on the cluster